### ISSUE TITLE
bug(a11y): Element has insufficient color contrast of 3.04 + no landmark

### ISSUE THREAD

**Issue by miclgael** [miclgael/issues.michaelgale.dev#5]
## Steps to reproduce the issue

1. Visit home-page using light theme.

## Expected result

![Screen Shot of blue element with black text](https://github.com/miclgael/issues/assets/4301358/547f2ac0-3100-4158-bf7d-8e6c79767e76)

## Actual result

![Screen Shot of blue element with white text](https://github.com/miclgael/issues/assets/4301358/2ae48957-ea3b-422f-a8b3-0959084de437)

## Notes

Update Deku theme config, change element to div and add a `role="region"`

**Comment by miclgael** [comment]
Work planned for 24/12/2023

**Comment by miclgael** [comment]
Fixed in release v3.5.22

**Comment by miclgael** [comment]
Fixed in release `v3.5.22`

**Comment by miclgael** [comment]
Fixed in app release `3.5.23` and Deku release `1.5.1`

**Comment by miclgael** [comment]
Fixed in app release `3.5.23` and Deku release `1.5.1`
